packages/cypress/BUILD.bazel

```diff
@@ -0,0 +1,99 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
+load("@build_bazel_rules_nodejs//:tools/defaults.bzl", "pkg_npm")
+load("@build_bazel_rules_nodejs//tools/stardoc:index.bzl", "stardoc")
+load("//third_party/github.com/bazelbuild/bazel-skylib:rules/copy_file.bzl", "copy_file")
+
+package(default_visibility = ["//visibility:public"])
+
+bzl_library(
+    name = "bzl",
+    srcs = glob(["**/*.bzl"]),
+    deps = [
+        "@build_bazel_rules_nodejs//:bzl",
+        "@build_bazel_rules_nodejs//internal/common:bzl",
+        "@build_bazel_rules_nodejs//internal/node:bzl",
+    ],
+)
+
+stardoc(
+    name = "docs",
+    testonly = True,
+    out = "index.md",
+    input = "index.bzl",
+    tags = ["fix-windows"],
+    deps = [":bzl"],
+)
+
+# Ugly genrule depending on local linux environment to build the README out of skylark doc generation.
+# Only referenced when we do a release.
+# TODO: This ought to be possible with stardoc alone. Need to coordinate with Chris Parsons.
+genrule(
+    name = "generate_README",
+    srcs = [
+        "index.md",
+        "install.md",
+    ],
+    outs = ["README.md"],
+    cmd = """cat $(execpath install.md) $(execpath index.md) | sed 's/^##/\\\n##/' > $@""",
+    tags = ["fix-windows"],
+    visibility = ["//docs:__pkg__"],
+)
+
+copy_file(
+    name = "npm_version_check",
+    src = "//internal:npm_version_check.js",
+    out = ":npm_version_check.js",
+)
+
+pkg_npm(
+    name = "npm_package",
+    srcs = [
+        "index.bzl",
+        "package.json",
+        "//packages/cypress:internal/cypress-install.js",
+        "//packages/cypress:internal/cypress_repository.bzl",
+        "//packages/cypress:internal/cypress_web_test.bzl",
+        "//packages/cypress:internal/plugins/base.js",
+        "//packages/cypress:internal/plugins/index.template.js",
+        "//packages/cypress:internal/run-cypress.js",
+    ],
+    build_file_content = """exports_files([
+    "internal/cypress-install.js",
+    "internal/cypress_repository.bzl",
+    "internal/cypress_web_test.bzl",
+    "internal/plugins/base.js",
+    "internal/plugins/index.template.js",
+    "internal/run-cypress.js",
+])""",
+    substitutions = {
+        "@build_bazel_rules_nodejs//packages/cypress:internal/cypress-install.js": "@npm//@bazel/cypress:internal/cypress-install.js",
```

Comment:
I suspect this can just be one entry
`//packages/cypress: //@bazel/cypress`

Reply:
I ended up needing to be specific since I have to do a different workspace for `cypress_repository.bzl`